examples/DrawingInTheAir/DrawingInTheAir_MotionActivated

```diff
@@ -0,0 +1,290 @@
+ * imagine you are writing on an invisible whiteboard, using your board as the
+ * pen), and the IMU data from these motions is used as training data for the
+ * PME. Once training is finished, you can keep drawing letters and the PME
+ * will try to guess which letter you are drawing.
+ *
+ * This example requires a button to be connected to digital pin 4
+ * https://www.arduino.cc/en/Tutorial/Button
+ *
+ * NOTE: For best results, draw big letters, at least 1-2 feet tall.
+ *
+ * Copyright (c) 2016 Intel Corporation.  All rights reserved.
+ * See license notice at end of file.
+ */
+
+#include "CurieIMU.h"
+#include "CuriePME.h"
+
+/*  This controls how many times a letter must be drawn during training.
+ *  Any higher than 4, and you may not have enough neurons for all 26 letters
+ *  of the alphabet. Lower than 4 means less work for you to train a letter,
+ *  but the PME may have a harder time classifying that letter. */
+const unsigned int trainingReps = 4;
+
+/* Increase this to 'A-Z' if you like-- it just takes a lot longer to train */
+const unsigned char trainingStart = 'A';
+const unsigned char trainingEnd = 'D';
+
+/* The input pin used to signal when a letter is being drawn- you'll
+ * need to make sure a button is attached to this pin */
+const unsigned int buttonPin = 4;
+
+/* Sample rate for accelerometer */
+const unsigned int sampleRateHZ = 200;
+
+/* No. of bytes that one neuron can hold */
+const unsigned int vectorNumBytes = 128;
+
+/* Number of processed samples (1 sample == accel x, y, z)
+ * that can fit inside a neuron */
+const unsigned int samplesPerVector = (vectorNumBytes / 3);
+
+/* This value is used to convert ASCII characters A-Z
+ * into decimal values 1-26, and back again. */
+const unsigned int upperStart = 0x40;
+
+const unsigned int sensorBufSize = 2048;
+const int IMULow = -32768;
+const int IMUHigh = 32767;
+
+boolean ledState = false;  // Motion Activation Flag 
+
+void setup()
+{
+    Serial.begin(9600);
+    while(!Serial);
+
+    pinMode(buttonPin, INPUT);
+
+    /* Start the IMU (Intertial Measurement Unit) */
+    CurieIMU.begin();
+CurieIMU.attachInterrupt(eventCallback);
+
+  /* Enable Zero Motion Detection */
+  CurieIMU.setDetectionThreshold(CURIE_IMU_ZERO_MOTION, 50);  // 50mg
+  CurieIMU.setDetectionDuration(CURIE_IMU_ZERO_MOTION, 2);    // 2s
+  CurieIMU.interrupts(CURIE_IMU_ZERO_MOTION);
+
+  /* Enable Motion Detection */
+  CurieIMU.setDetectionThreshold(CURIE_IMU_MOTION, 20);      // 20mg
+  CurieIMU.setDetectionDuration(CURIE_IMU_MOTION, 10);       // trigger times of consecutive slope data points
+  CurieIMU.interrupts(CURIE_IMU_MOTION);
+
+  Serial.println("IMU initialisation complete, waiting for events...");
+    /* Start the PME (Pattern Matching Engine) */
+    CuriePME.begin();
+
+    CurieIMU.setAccelerometerRate(sampleRateHZ);
+    CurieIMU.setAccelerometerRange(2);
+pinMode(11, OUTPUT);
+digitalWrite(11, LOW);
```

Comment:
Give the pin number a name, e.g. "LED_PIN" or whatever, instead of writing "11" everywhere